docker-compose.yml

```diff
@@ -38,6 +38,11 @@ services:
       MECA_SFTP_USERNAME: 'test'
       MECA_SFTP_PASSWORD: 'test'
       MECA_SFTP_PORT: 22
+      SEND_MAIL: 'false'
```

Comment:
We'll need to create some form of infrastructure issue to add these to the the `reviewer` charts. Would be good to go through this again with @erkannt so we know how to go about adding the values to the deployment config too.

Reply:
https://github.com/libero/reviewer/pull/1205/files 

Just seen this. So its just the deployment values that need sorting for now then